semver.py

```diff
@@ -344,6 +386,31 @@ def isvalid(cls, version):
         except ValueError:
             return False
 
+    @staticmethod
+    def format_version(major, minor, patch, prerelease=None, build=None):
```

Comment:
Does this really need to be a public method? I don't find it particularly useful; I'd just inline it in `__str__` and suggest `str(VersionInfo(...))` as substitute of the deprecated `format_version` function.

Reply:
Good point! :+1: I removed the above function and added the `str(VersionInfo(...))` fragment into the original module level function `format_version`.